types/next/document.d.ts

```diff
@@ -64,6 +64,7 @@ export interface DocumentProps<Q extends DefaultQuery = DefaultQuery> {
     files: string[];
     dynamicImports: string[];
     assetPrefix: string;
+    styleTags: string[];
```

Comment:
DocumentProps takes in a generic, we can declare a type that can be used in DocumentProps. Don't think we need this

Reply:
Yep. You right